addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1627,8 +1627,14 @@ body.editor_enable.editor_has_snippets {
                 background-color: $o-we-sidebar-content-fold-block-bg;
                 box-shadow: $o-we-item-standup-top rgba($o-we-item-standup-color-dark, .5), $o-we-item-standup-bottom rgba($o-we-item-standup-color-light, .2);
 
-                we-collapse.active, we-collapse.active .o_we_collapse_toggler {
+                we-collapse.active {
                     background-color: $o-we-bg-lighter;
+                    we-toggler.o_we_collapse_toggler {
+                        background-color: $o-we-bg-lighter;
+                    }
+                    we-collapse.active, we-collapse.active .o_we_collapse_toggler {
+                        background-color: $o-we-sidebar-content-fold-block-bg;
+                    }
```

Comment:
> No it cannot, it was so that the rule for the background-color would affect it as well. Maybe the current change is better?

Better indeed but still not ok. Of course you don't want to create a selector which matches DOM which cannot exist when you can so your change does that well. However, you still have a rule saying "set lighter background"... which is duplicated for no reason. The code should also be factorized (like it should have been before). So right now you have

```scss
A {
    rule1;

    B {
        rule1;
    }

    A, A C {
        rule2;
    }
}
```
You can do:
```scss
A {
    &, B {
        rule1;
    }
    A {
        &, C {
            rule2;
        }
    }
}
```
That has 3 effects:
- You don't duplicate the SCSS code of rule1
- You create only one CSS rule for rule 1 (with two selectors still but one rule)
- You don't repeat A twice for rule 2 (no effect on CSS but "better" SCSS)